components/globe-component/globe-component.js

```diff
@@ -191,17 +182,43 @@ export default {
       this.mouse.x = ((event.clientX / this.renderer.domElement.clientWidth) * 2) - 1
       this.mouse.y = -((event.clientY / this.renderer.domElement.clientHeight) * 2) + 1
     },
-    /**
-     * Pan to the active story
-     */
-    pan () {
-      console.log('panning to', this.activeStory)
+    panToActiveMarker () {
+      if (this.activeMarker && this.activeMarker.location) {
+        const from = cartesian2polar(this.camera.position.x, this.camera.position.y, this.camera.position.z)
+        from.lat = θ2deg(from.θ)
+        from.lng = φ2deg(from.φ)
+
+        // create a to object
+        const to = {}
+        // clone object
+        Object.assign(to, this.activeMarker.location)
```

Comment:
We have to remember to polyfill this one

Reply:
I would normally use _.clone from lodash. I see it's not supported by IE. Replaced by explicit assignment.